README.md

```diff
@@ -21,16 +21,59 @@ A set of services used by [Blockscout](https://blockscout.com/) blockchain explo
 
 ## Services
 
-1. [smart-contract-verifier](smart-contract-verifier/) - provides API for ethereum contract verification written in Solidity and Vyper
+1. [blockscout-ens](blockscout-ens) - indexed data of domain name service for blockscout instances.
+
+1. [da-indexer](da-indexer) - collects blobs from different DA solutions (e.g, Celestia) 
+
+2. [eth-bytecode-db](eth-bytecode-db/) - Ethereum Bytecode Database. Cross-chain smart-contracts database used for automatic contracts verification
+
+1. [proxy-verifier](proxy-verifier) - backend for the standalone multi-chain verification service
+
+1. [scoutcloud](scoutcloud) - API to deploy and manage blockscout instances
 
 2. [sig-provider](sig-provider/) - aggregator of ethereum signatures for transactions and events
 
-3. [multichain-search](multichain-search/) - service for searching transactions, blocks, tokens, etc in all blockscout instances. Contains frontend and backend parts.
+3. [smart-contract-verifier](smart-contract-verifier/) - smart-contracts verification
+
+1. [stats](stats) - service designed to calculate and present statistical information from a Blockscout instance
+
+1. [user-ops-indexer](user-ops-indexer) - service designed to index, decode and serve user operations as per the ERC-4337 standard
 
 4. [visualizer](visualizer/) - service for evm visualization such as:
    1. Solidity contract visualization using [sol2uml](https://www.npmjs.com/package/sol2uml)
 
-5. [eth-bytecode-db](eth-bytecode-db/) - Ethereum Bytecode Database. Verifies smart-contracts and searches for the sources via bytecodes  
+## Running and configuring services
+
+Services are distributed as docker images. For each service you can find information about packages and their recent releases
+inside service directories.
+
+You can configure your app by passing necessary environment variables when starting the container. 
+Configuration variables common for all services can be found [here](docs/common-envs.md).
+See full list of ENVs and their description inside service directories.
+
+```shell
+docker run -p 8050:8050 --env-file <path-to-your-env-file> ghcr.io/blockscout/{service-name}:latest 
+```
+
+Alternatively, you can build your own docker images or compile them directly from sources. 
+Some of such options are described in [build](docs/build.md).
+
+## Project Layouts
```

Comment:
maybe also mention `justfile`s that are quite useful & present in most of the services